src/collapse/mod.rs

```diff
@@ -1,3 +1,10 @@
+/// Stack collapsing for the output of [`dtrace`](...).
```

Comment:
We should probably have a link for this. [Joyent's page](https://www.joyent.com/dtrace) is probably the right one to use. We could also be helpful and include a link to @brendangregg's [dtrace tools page](http://www.brendangregg.com/dtrace.html).

Reply:
👍